### PR TITLE
Wok 503 dokumentation + facet endpoint 

### PR DESCRIPTION
**Pull request by VictorHarbo**
2-in-1 pull request. 

First part is only documentation
Second part relates to the aviser/query/facet endpoint, which did not make use of the startTime and endTime parameters. This has been fixed, however I would like to know if I have fixed it the right way or by hacking it.

### CODE REVIEW

File: src/main/openapi/openapi.yaml [hunk 1]
```diff
@@ -538,19 +540,19 @@ components:
...
         titleUUID:
-          description: 'TODO: Explain this'
+          description: 'The ID for the title of the newspaper, which the article is from.'
           type: string
         familyId:
-          description: 'TODO: Explain this'
+          description: 'The general name of the newspaper. The name of newspapers can change over time, this familyId will always be the same eventhough the title of the newspaper changes a little'
           type: string
         newspaper_page:
           description: 'The page number of the addition that the article appears on'
```
Comment (tokee) [inline src/main/openapi/openapi.yaml:549]: `edition`

File: src/main/openapi/openapi.yaml [hunk 2]
```diff
@@ -538,19 +540,19 @@ components:
           description: 'Human readable version of the edition'
           type: string
         titleUUID:
-          description: 'TODO: Explain this'
+          description: 'The ID for the title of the newspaper, which the article is from.'
           type: string
         familyId:
-          description: 'TODO: Explain this'
+          description: 'The general name of the newspaper. The name of newspapers can change over time, this familyId will always be the same eventhough the title of the newspaper changes a little'
```
Comment (tokee) [inline src/main/openapi/openapi.yaml:546]: Maybe you could add examples for all the fields? Even with explanations some of them are a bit cryptic. You could make a search for an old (>140 years) article and simple copy-paste from that.

**Review by tokee — CHANGES_REQUESTED** [review]
Overall it's great to have better descriptions. My two significant requests are for aa small code change and for examples of values in the `components`.

File: src/main/java/dk/kb/labsapi/SolrExport.java [hunk 3]
```diff
@@ -271,14 +275,13 @@ public static FACET_SORT getDefault() {
...
-            FACET_FORMAT outFormat) {
+            FACET_FORMAT outFormat) throws ParseException {
 
-        // TODO: Why are start and end not used?
         String trueStartTime = ParamUtil.parseTimeYearMonth(startTime, minYear, minYear, maxYear, true);
         String trueEndTime = ParamUtil.parseTimeYearMonth(endTime, maxYear, minYear, maxYear, false);
 
         SolrParams request = new SolrQuery(
-                CommonParams.Q, sanitize(query),
+                CommonParams.Q, sanitize(query + " AND timestamp:[" + trueStartTime + " TO " + trueEndTime + "]"),
```
[the remark this answers is not in the document]
Comment (VictorHarbo) [reply src/main/java/dk/kb/labsapi/SolrExport.java]: Temporarily fixed - added parenthesis around query

File: src/main/openapi/openapi.yaml (hunk 1 above)
Comment (VictorHarbo) [reply src/main/openapi/openapi.yaml:549]: I am not quite sure where  you want edition to be mentioned in this line? Can you please elaborate?

**Review by tokee — APPROVED** [review]
LGTM

**Pull request merged**